Formula/sqlpage.rb

```diff
@@ -0,0 +1,20 @@
+class Sqlpage < Formula
+  desc "Web application framework, for creation of websites with simple database queries"
+  homepage "https://sql.ophir.dev/"
+  url "https://github.com/lovasoa/SQLpage/archive/refs/tags/v0.7.2.tar.gz"
+  sha256 "e6f7e0fa0d96e13560f7a010e9d1df0184a2bb610f70311647ffdecea9ba671a"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    pid = spawn("sqlpage", [:out, :err]=>["log", "w"])
+    sleep(2)
+    assert_includes File.read("log"), "Starting server on 0.0.0.0:8080"
+    Process.kill(9, pid)
```

Comment:
can we do something like this instead?
```suggestion
    port = free_port
    pid = fork do
      exec "sqlpage", "--port", port.to_s
    end
    sleep(2)
    assert_match "hello", shell_output("curl http://localhost:#{port}")
    Process.kill(9, pid)
```